App/State/ExposureDetectionState.swift

```diff
@@ -28,6 +28,9 @@ struct ExposureDetectionState: Codable {
   /// They are stored so that they can be sent in case of a Data Upload (see `Logic.DataUpload`)
   var recentPositiveExposureResults: [PositiveExposureResult] = []
 
+  /// The list contains the selected country, the selection date and the latestProcessedKeyChunkIndex of the country
+  var countriesOfInterest: [CountryOfInterest] = []
```

Comment:
As mentioned above, I would use a dictionary here instead. There is no ordering involved and a dictionary would better fit the main use cases of this variable